Common/UI/ContentSwitcher/content_switcher.gd

```diff
@@ -2,15 +2,17 @@ extends Control
 
 signal Switched(scn: Node)
 
-@export var current_scene: Node
+@onready var current_scene = $Home
```

Comment:
This works for the main menu when `$home` exists, but throws an error on the level select screen where `$home` does not exist.
The commented line is probably the better method. We set the first current scene from outside of the content switcher  instead of from within this script (see the other comment).

This means that we don't have to preload every possible current scene from within `content_switcher`. So far it would only be the `home` and `level_select` scenes, but this will grow over time.

Reply:
If you can take your time on this matter, sure you can make the change by yourself !
I'll do all I can to make a good battle stat system 🔥

Reply:
I'll do that, thanks!